sky/shell/platform/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h"
+#include "sky/shell/platform/ios/framework/Headers/FlutterViewController.h"
+
+@implementation FlutterAppDelegate {
+  UIWindow* _window;
```

Comment:
Please `@synthesize` the window property. This is required for conformance to the `UIApplicationDelegate` protocol that `FlutterAppDelegate` denotes it conforms to.